components/service/glean/src/test/java/mozilla/components/service/glean/storages/EventsStorageEngineTest.kt

```diff
@@ -196,30 +195,36 @@ class EventsStorageEngineTest {
             allowedExtraKeys = listOf("test_event_number")
         )
 
-        resetGlean(getContextWithMockedInfo(), Glean.configuration.copy(
-            serverEndpoint = "http://" + server.hostName + ":" + server.port,
-            logPings = true
-        ))
+        resetGlean(getContextWithMockedInfo(), Glean.configuration.copy(logPings = true))
 
-        try {
-            // We send 510 events.  We expect to get the first 500 in the ping, and 10 remaining afterward
-            for (i in 0..509) {
+        // We send 510 events.  We expect to get the first 500 in the ping, and 10 remaining afterward
+        for (i in 0..509) {
+            runBlocking {
```

Comment:
Trying to make the one flaky test I had run properly.

Reply:
Does it work without it? Because this `runBlocking` is not blocking the right dispatcher, as far as I can tell, so it shouldn't do anything, as far as I understand.